ddl/job_table.go

```diff
@@ -203,7 +203,7 @@ func (d *ddl) processJobDuringUpgrade(sess *sess.Session, job *model.Job) (isRun
 	if job.IsPausedBySystem() && !hasSysDB(job) {
 		var errs []error
 		errs, err = ResumeJobsBySystem(sess.Session(), []int64{job.ID})
-		if len(errs) > 0 {
+		if len(errs) > 0 && errs[0] != nil {
```

Comment:
Is it possible to have `errs` like `[nil, "some err", nil]`?

Reply:
Yes.

Reply:
@tangenta PTAL